src/test/java/pl/itacademy/schedule/generator/ScheduleGeneratorTest.java

```diff
@@ -38,4 +39,22 @@ public void generate_generatesScheduleFromParameters() {
         assertThat(schedule.getLessons(), containsInAnyOrder(firstLesson, secondLesson));
     }
 
+    @Test
+    public void generate_generatesScheduleFromParametersWithWarning() {
```

Comment:
Actually it there are we have no warnings in method.
Probably test name should be different:
generate_requiredHoursCantBeFitIntoDailySchedule_createsLastDayShorer_addsFlagForShorterLastDay
or something like this.

Reply:
OK